deploy/charts/library/templates/_cluster-role.tpl

```diff
@@ -70,9 +70,15 @@ rules:
   - apiGroups:
       - ceph.rook.io
     resources:
-      - "*"
```

Comment:
`cephnfses` is required also. 

As a larger note, this will disable a lot of features of the orchestrator with the exception of clusters and clients. What need does the mgr have for modifying cephclients and cephclusters?

@travisn thoughts?

Reply:
> As a larger note, this will disable a lot of features of the orchestrator 
Please @BlaineEXE can you elaborate, or do we need to update to all the CRDs for making it not block anything?

Reply:
@parth-gr Just like all CRDs were added in place of `*` in other rbac, let's add all CRDs to the list here too.